pensjon-brevbaker/src/main/resources/latex/firstpage.tex

```diff
@@ -0,0 +1,95 @@
+\fancypagestyle{firstpage}{
+    \fancyhf{}%
+    \renewcommand{\headrulewidth}{0pt}%
+    \returnaddress
+    \logo
+    \toaddress
+    \letterdate
+    \fancyfoot[C]{\letterpagenumber}
+    \fancyfoot[L]{\firstfooter}
+}
+
+\newcommand{\firstpage}{
+    \enlargethispage{-2cm} % reduce textheight to not overlap with footer on the first page.
+    \thispagestyle{firstpage} %set pagestyle for this page
+    \pagestyle{normalpage} % set default pagestyle
+}
+
+
+%----------------------------------------
+%         heading / return address
+%----------------------------------------
+\newcommand{\returnaddress}{
+    \begin{textblock*}{140mm}(20mm,18.5mm)
+        \parskip=0pt
+        \baselineskip=5pt
+        \tinyText{\feltreturadresseenhetprefix \space \feltnavenhet}\newline
+        \tinyText{\feltreturadresse \space \feltreturadressepostnrsted}
+    \end{textblock*}
+}
+
+
+%----------------------------------------
+%              To address
+%----------------------------------------
+\newcommand{\toaddress}{
+    \begin{textblock*}{100mm}(20mm,55.7mm)
+        \secondaryFont \fontsize{12pt}{12pt} \selectfont
+        \feltmottakeradresselineone \newline
+        \feltmottakeradresselinetwo \newline
+        \feltmottakeradresselinethree \newline
+        \feltmottakeradresselinefour \newline
+        \feltmottakeradresselinefive
+    \end{textblock*}
+}
+
+
+%----------------------------------------
+%                  Date
+%----------------------------------------
+
+\newcommand{\letterdate}{
+    \begin{textblock*}{40mm}(160mm,86mm)\tinyText{\feltdatoprefix}\primaryFont\fontsize{12pt}{12pt}\selectfont~\feltdato\end{textblock*}
+}
+
+%----------------------------------------
+%           Title and opening
+%----------------------------------------
+
+%Title font
+\newcommand{\titleText}[1]{\textbf{\primaryFont \fontsize{16pt}{16pt} \selectfont \newline #1}}
+
+%opening prefix font
+\newcommand{\tinyGrayText}[1]{{\grayFont \fontsize{8pt}{8pt} \selectfont #1}}
+
+\newcommand{\sakspart}{
```

Comment:
Du har plassert `\sakspart` kommandoen i firstpage.tex, regner med at det ikke betyr at den ikke kan brukes i vedlegg? Den brukes nemlig i OMSORG_EGEN_AUTO vedlegget.

Reply:
Flyttet sakspart til pensjonsbrev_V3.cls
ligger en sak på å tilpasse mastermalen for ulike brev-typer:
https://jira.adeo.no/browse/PL-4647